code/modules/hydroponics/grown/mushrooms.dm

```diff
@@ -199,6 +231,31 @@
 	icon_state = "chanterelle"
 	filling_color = "#FFA500"
 
+//Truffles
+/obj/item/seeds/chanter/truffles
+	name = "pack of truffle mycelium"
+	desc = "A staple of good cuisine."
+	icon_state = "mycelium-truffles"
+	species = "truffles"
+	plantname = "Truffles"
+	product = /obj/item/reagent_containers/food/snacks/grown/mushroom/truffle
+	lifespan = 35
+	endurance = 20
+	maturation = 10
+	production = 12
+	yield = 1
+	potency = 5
+	growthstages = 2
+	genes = list(/datum/plant_gene/trait/plant_type/fungal_metabolism)
+	growing_icon = 'icons/obj/hydroponics/growing_mushrooms.dmi'
+	reagents_add = list(/datum/reagent/consumable/nutriment = 0.1,/datum/reagent/consumable/pricy=0.05)
```

Comment:
I think you meant to set the pricy reagent to the truffle_oil reagent, also you should probably make truffle_oil in food_reagents not drink_reagents

Reply:
what is a "pricy"?